x/auth/ante/setup.go

```diff
@@ -51,7 +51,7 @@ func (sud SetUpContextDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 			switch rType := r.(type) {
 			case sdk.ErrorOutOfGas:
 				log := fmt.Sprintf(
-					"out of gas in location: %v; gasWanted: %d, gasUsed: %d",
+					"insufficient gas\n  gasOffered: %d, gasRequired: %d \n  code location: %v",
```

Comment:
OK to change the string, but I feel not so comfortable putting `\n` in an error string. How does this render e.g. on CLIs?

The problem described in the PR's discussion also seems like a pure UI problem.

```suggestion
					"insufficient gas, gasOffered: %d, gasRequired: %d, code location: %v",
```

Reply:
agreed that it could be fixed in all the derivatives where it shows up. and I do think that dApps like Osmosis should handle errors from the sdk more carefully...
but still it is better to use words that more accurately describe the situation... offered and required feel much better and understandable than wanted and used....